src/networks.json

```diff
@@ -2190,5 +2190,21 @@
     },
     "start": 501318,
     "logo": "ipfs://QmYz8LK5WkUN8UwqKfWUjnyLuYqQZWihT7J766YXft4TSy"
+  },
+  "1116": {
+    "key": "1116",
+    "name": "Core Chain Mainnet",
+    "shortName": "Core",
+    "chainId": 1116,
+    "network": "mainnet",
+    "multicall": "0x024f0041b76B598c2A0a75004F8447FaF67BD004",
+    "rpc": [
+      "https://rpceuwest3.coredao.org"
```

Comment:
Hi @leeyanbtcs Looks like this is not a full archive node? i get `missing trie node a705014fcbd00e291767bda7479b41ecdd9b5b20604d061e19acb01b95a9153d` error